app/routes/LockingConfiguration/components/Layout.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+import Page from 'components/Page'
+import styles from './index.css'
+
+class Layout extends Component {
+
+  render() {
+    const {
+      minutes,
+      handleOptionChange,
+      handleSaveConfig,
+    } = this.props
+
+    return (
+      <Page account logOut>
+        <form>
+          <div className={styles.item}>
```

Comment:
```
const TimeBlock = ({minTime, onOptionChange, }) => (
    <div className={styles.item}>
        <input type='radio' value={`${minTime}`}  checked={minutes === minTime} onChange={onOptionChange} />
        {minTime}min {minTime === 5 && `(default)`}
    </div>
)
```
And then you can reuse this component, instead of having it repeated 6 times.